components/fieldgroups/account/events/account-data-value-change.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/account/events/data-value-change",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Data Value Change",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "dataValueChange.accountDataValueChange",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/account-history"
+  ],
+  "description": "Use to capture history of a change in an account attribute.",
+  "definitions": {
+    "accountDataValueChange": {
+      "properties": {
+        "xdm:dataValueChange": {
+          "title": "Data Value Change",
+          "type": "object",
+          "properties": {
+            "xdm:attributeName": {
+              "title": "Attribute Name",
+              "type": "string",
+              "meta:titleId": "data-value-change##xdm:attributeName##title##73041",
+              "meta:descriptionId": "data-value-change##xdm:attributeName##description##10571"
+            },
+            "xdm:reason": {
+              "title": "Reason",
+              "type": "string",
+              "meta:titleId": "data-value-change##xdm:reason##title##73041",
+              "meta:descriptionId": "data-value-change##xdm:reason##description##10571"
+            },
+            "xdm:source": {
+              "title": "Source",
+              "type": "string",
+              "meta:titleId": "data-value-change##xdm:source##title##73041",
+              "meta:descriptionId": "data-value-change##xdm:source##description##10571"
+            },
+            "xdm:oldValueString": {
+              "title": "Old Value String",
+              "type": "string",
+              "description": "Old value.",
```

Comment:
We should include the description that this is the old value for String fields, number fields, datetime fields etc. Applies to all places following this.

Reply:
Adjusted the descriptions to say string, number or date